src/FilterButtonList.js

```diff
@@ -11,10 +10,10 @@ import {
 } from './layout'
 import DefaultIcon from './DefaultIcon'
 import DefaultMissingTypeComponent from './DefaultMissingTypeComponent'
-import InjectTreeNode from './utils/injectTreeNode'
+import { withNode, withLoader } from './utils/injectTreeNode'
```

Comment:
I was using these components for testing anyway, so I figured I'd include the changes in the initial PR as a demo of what the new usage would look like